.github/workflows/reusable-porting-guide.yml

```diff
@@ -0,0 +1,92 @@
+---
+name: Create porting guide PR
+
+"on":
+  workflow_call:
+    inputs:
+      repository:
+        type: string
+        description: The repository to check out.
+        required: true
+      path:
+        type: string
+        description: The path for the repository.
+        required: true
+      ansible-version:
+        type: string
+        description: Ansible release version.
+        required: true
+      ansible-major-version:
+        type: string
+        description: Ansible major version.
+        required: true
+
+env:
+  GIT_BRANCH: porting-guide-${{ inputs.ansible-version }}
+  GITHUB_ACTOR: ${{ github.actor }}
```

Comment:
This is already populated automatically: https://docs.github.com/en/actions/how-tos/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables
```suggestion
```